articles/azure-app-configuration/faq.md

```diff
@@ -100,7 +100,7 @@ You can't downgrade a store from the Standard tier to the Free tier. You can cre
 
 ## Are there any limits on the number of requests made to App Configuration?
 
-Configuration stores in the Free tier are limited to 1,000 requests per day. Configuration stores in the Standard tier may experience temporary throttling when the request rate exceeds 20,000 requests per hour.
+In App Configuration, when reading key-values, data will be paginated and each request can read up to 100 key-values. When writing key-values, each request can create or update one key-value. Configuration stores in the Free tier are limited to 1,000 requests per day. Configuration stores in the Standard tier may experience temporary throttling when the request rate exceeds 20,000 requests per hour.
```

Comment:
I know this is not the line you changed but for the sentence about standard tier, we should mention that the number of requests is unlimited and the first 200k are included. The way there sentences are written it would be easy to think that the limit is 1k for the free tier and 20k for the standard tier

Reply:
I agree if people only read this one, but we have another Q above this one. It explained the 200k and cost in more detail, so I think we should be fine not repeating that here.